TetrisDotnet/Code/Events/EventType.cs

```diff
@@ -0,0 +1,16 @@
+namespace TetrisDotnet.Code.Events
+{
+    public enum EventType
+    {
+        // Input Events
+        InputRotateClockwise,
```

Comment:
doing this would mean parsing the keycode in the receiver, and the receiver would need to have the knowledge of which keycode does what (example: determine that W and UpArrow mean rotate clockwise). Would reduce event types, but would increase code, management and complexity on the receiving end.